backend/core/package.json

```diff
@@ -25,11 +25,15 @@
     "@nestjs/cli": "^7.0.0",
     "@nestjs/common": "^7.1.0",
     "@nestjs/core": "^7.1.0",
+    "@nestjs/passport": "^7.0.0",
     "@nestjs/platform-express": "^7.1.0",
     "@nestjs/typeorm": "^7.1.0",
+    "bcrypt": "^4.0.1",
```

Comment:
Can you take a quick look at using `scrypt` instead of `bcrypt`? My understanding is that it's generally preferred from a security perspective these days ([e.g. here](https://latacora.singles/2018/04/03/cryptographic-right-answers.html)), and it's built into Node 10+ so it would save us an external package to keep current in a security sensitive (if admittedly pretty stable) place.

I'm ok with punting it to a subsequent PR if it's somehow a more difficult switch than I think. bcrypt isn't all that bad, but I'd just like to save us all the migration logic later.

Reply:
I think it's worth making this choice _before_ we have data rather than migrating later ;-). Migrating hashes is pretty problematic for the obvious reason that we don't actually have the original PW. 

I can use `scrypt`, it's _slightly_ more effort that `bcrypt` because you have to save your `salt` as well to verify, but I can just concat that onto the `hash` string itself just like `bcrypt` does internally.